src/js/utils/dom.js

```diff
@@ -9,6 +9,25 @@ import window from 'global/window';
 import document from 'global/document';
 import mejs from '../core/mejs';
 
+
+// Cuz ie sucks
+const TinyPromise = (h,f=[],b=-1,g,c,l) => (
```

Comment:
Can you put this uncompressed, please? It's difficult to understand what this method does